its/projects/connectedmode-json/pom.xml

```diff
@@ -0,0 +1,29 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.sonarsource.sonarlint.core</groupId>
+  <artifactId>connectedmode-json</artifactId>
```

Comment:
I think `connectedmode-json` is not a super great name; what about `shared-connected-mode`, for instance? Or something else